### ISSUE TITLE
associated and quoted folios in ann_046_fa_16_LePouesard_PainOstieRostieBreadInEarlyModernEurope

### ISSUE THREAD

**Issue by njr2128** [cu-mkp/m-k-manuscript-data#1487]
Opening this issue to deal with a slightly more complicated issue that arose as a result of #1477.

A reference to "rostie" is found on 123v as part of a continued entry from 123r (entry id = 123r_1) which was previously unaccounted for.

From GD comments left in abstract doc, 
>PHS: Ok, I've entered pg #s into abstract (and in the process discovered another instance of "rostie"--on a cont'd page that must not have been found in Emma's search!!). If we can also link to 123v, that would be great, but it is cont'd, so not a heading. If we cannot, it doesn't matter. I've added 123v to the text and to the table in the essay and have linked everything (and corrected bibliog and fn ann cite).

The full text now references and links to 123v. The table that begins the essay with a summary of instances has a reference to 123v with a "recipe title" I think provided by PHS. This is rather than listing 123r and the heading "A means di far correr lotnegra."

I think that in order to follow our protocol, the essay should be linked to 123r_1 (the entry id), which means that in folio view, it will show up on 123r since that is where the heading is. So while the instance is technically on 123v, it should be associated with 123r. 

In the full text of the essay, I think we could keep links to 123v, but we need to explain that the entry begins on 123r and cite it properly as 123r. I think this should just be clearly explained to the reader that it is a continued entry that begins on 123r but that the instance of "rostie" occurs on 123v.

@thuchacz and @ps2270 - can you advise?
In short, I propose:
- adding 123_1 to referenced folios in annotations-metadata so that it is associated accordingly
- changing the table at the beg of the essay text to cite 123r and the heading properly, with a note that "rostie" is actually on the continued part on 123v.
- elsewhere in the essay text, we can leave as is

**Comment by ps2270** [comment]
In short, I propose:  I AGREE! This is fine.

   - adding 123_1 to referenced folios in annotations-metadata so that it is associated accordingly
   - changing the table at the beg of the essay text to cite 123r and the heading properly, with a note that "rostie" is actually on the continued part on 123v.
    - elsewhere in the essay text, we can leave as is

**Comment by ps2270** [comment]
I completed 2 and 3 above and will now do 1.

**Comment by njr2128** [comment]
folio references have been updated in annotations-metadata
